Doc/library/turtle.rst

```diff
@@ -213,6 +213,32 @@ useful when working with learners for whom typing is not a skill.
     use turtle graphics with a learner.
 
 
+Automatically begin and end filling
+-----------------------------------
+
+If you have Python 3.14 or later, you don't need to call :func:`begin_fill` and
+:func:`end_fill` for filling. Instead, you can use the :func:`fill`
+:term:`context manager` to automatically begin and end fill. Here is an
+example::
+
+   with fill():
+      for i in range(4):
+         forward(100)
+         right(90)
+
+   forward(200)
```

Comment:
Can you use 4-space for the inner code please? The code block should indeed use 3-space (you can even use `.. code-block:: python` so that IDEs render the inner block) but `for` should be indented by 4 spaces.